test/WebJobs.Script.Tests/Rpc/LanguageWorkerChannelTests.cs

```diff
@@ -56,6 +56,24 @@ public LanguageWorkerChannelTests()
                0);
         }
 
+        [Fact]
+        public void Dispose_InvokesWaitForTimeoutOrBuffer()
```

Comment:
Expand this test 
- Set up invocation buffers
- Send invocation responses
- Verify dispose waithandle is set once all the invocation responses are received.